lib/Doctrine/Locator.php

```diff
@@ -167,8 +167,6 @@ public function locate($name)
 
             return $this->_resources[$name];
         }
-
-        throw new Doctrine_Locator_Exception("Couldn't locate resource " . $name);
```

Comment:
This was removed because it's unreachable (both paths of the if condition above return).